src/resolvers/admin.ts

```diff
@@ -23,4 +24,20 @@ export class AdminResolver {
     await killSwitch.save()
     return { isKilled }
   }
+
+  @Mutation(() => GeneratedSignature)
+  async generateSignature(@Arg('bytes', () => String) bytes: string) {
+    if (!config.appPrivateKey) {
+      throw new Error('No PRIVATE_KEY provided to generate signature')
+    }
+
+    if (!cryptoIsReady()) {
```

Comment:
What does this actually mean, what do we need to wait for? Maybe it would be best to make sure it's ready during Orion startup?

Reply:
It's checking whether the bridge between JS and WASM was instantiated. Good idea.